src/service/comparison_vs_peers/comparison_vs_peers_service.py

```diff
@@ -11,6 +11,10 @@ def remove_base_metrics(self, company: dict) -> None:
             "prior_actuals_revenue",
             "budget_revenue",
             "budget_ebitda",
+            "actuals_cost_of_goods",
+            "actuals_sales_of_marketing",
+            "actuals_general_and_admin",
+            "actuals_research_and_development",
```

Comment:
What about budget values?
We should only have the metrics for the reports, if we don't delete all unnecessary metrics, the download button won't work

Reply:
budget values were added.